### ISSUE TITLE
Publish Erlang docs generated with ExDoc to Hexdocs

### ISSUE THREAD

**Issue by DaniruKun** [erlang/otp#5516]
**Is your feature request related to a problem? Please describe.**
Newcomers to Elixir often find it strange that they need to switch to a different source of information entirely for certain Erlang module and function documentation. Not to mention while the Edoc generated docs have become better recently, they are still not nearly as user friendly as ExDoc.
Another reason raised recently in developer tooling was the generation of hyperlinks to external doc sources. If both Erlang and Elixir core library functions follow the same resource path conventions, it is easier to handle.
**Describe the solution you'd like**
1. Generate [ex_doc](https://github.com/elixir-lang/ex_doc) from Erlang edoc tags in parallel
2. Publish the generated docs either to a central doc repository like HexDocs.pm, or self-host on e.g. Github Pages.
**Describe alternatives you've considered**

**Additional context**

**Comment by garazdawi** [comment]
Hello!

There is an ongoing effort by the Erlang Ecosystem Foundation to improve things in this area. There used to be a Documentation working group, but that was merged into the Build and Packaging working group.

You can track the progress in [their github repo](https://github.com/erlef/build-and-packaging-wg/issues). The specific issues related to this would be https://github.com/erlef/build-and-packaging-wg/issues/24 and https://github.com/erlef/build-and-packaging-wg/issues/25.

I also know that [Wojtek has been working](https://github.com/elixir-lang/ex_doc/pull/1291#issuecomment-773584253) on generating ExDoc from our [EEP-48 style doc chunks](https://www.erlang.org/doc/apps/kernel/eep48_chapter.html), but I haven't heard anything about that in a while.

So things are moving in that direction, and I think the best way to help out would be to contribute to ExDoc so that it can handle the Erlang format for EEP-48 style documentation.

**Comment by erszcz** [comment]
Indeed, as mentioned in https://github.com/erlef/build-and-packaging-wg/issues/25#issuecomment-1004932870 [`rebar3_hex`](https://hexdocs.pm/rebar3_hex/readme.html) is capable of publishing Erlang project documentation generated by ExDoc to hexdocs.pm. This is already happening in the community as evidenced by projects such as `rebar3_hex` itself, but also [`telemetry`](https://hexdocs.pm/telemetry/readme.html), [`systemd`](https://hexdocs.pm/systemd), [`ram`](https://hexdocs.pm/ram/readme.html), or [`syn`](https://hexdocs.pm/syn/readme.html) to name a few.

@DaniruKun is that what you had in mind?

**Comment by DaniruKun** [comment]
> Indeed, as mentioned in [erlef/build-and-packaging-wg#25 (comment)](https://github.com/erlef/build-and-packaging-wg/issues/25#issuecomment-1004932870) [`rebar3_hex`](https://hexdocs.pm/rebar3_hex/readme.html) is capable of publishing Erlang project documentation generated by ExDoc to hexdocs.pm. This is already happening in the community as evidenced by projects such as `rebar3_hex` itself, but also [`telemetry`](https://hexdocs.pm/telemetry/readme.html), [`systemd`](https://hexdocs.pm/systemd), [`ram`](https://hexdocs.pm/ram/readme.html), or [`syn`](https://hexdocs.pm/syn/readme.html) to name a few.
> 
> @DaniruKun is that what you had in mind?

Yes, this is exactly it, thank you for your work! 👍
